votingpool/pool.go

```diff
@@ -290,47 +339,75 @@ func (vp *Pool) putSeries(version, seriesID, reqSigs uint32, inRawPubKeys []stri
 //
 // - rawPubKeys has to contain three or more public keys;
 // - reqSigs has to be less or equal than the number of public keys in rawPubKeys.
-func (vp *Pool) CreateSeries(version, seriesID, reqSigs uint32, rawPubKeys []string) error {
-	if series := vp.GetSeries(seriesID); series != nil {
+// FIXME: The minimum allowed value for seriesID should be 1:
```

Comment:
This should probably be done before it goes in.  It seems pretty straight forward.